sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/ParquetVectorUpdaterFactory.java

```diff
@@ -195,12 +203,11 @@ boolean isTimestampTypeMatched(LogicalTypeAnnotation.TimeUnit unit) {
       annotation.getUnit() == unit;
   }
 
-  void validateTimestampType(DataType sparkType) {
+  void validateTimestampNTZType(DataType sparkType) {
```

Comment:
1. The input parameter `sparkType` is no longer in use, can it be removed?
2. Although the old code is like this, in non-ea mode, `assert` is invalid.
3. Is it possible to reduce the access scope of the `validateTimestampNTZType` method to `private` in this pr? It does not need to be accessed by other classes.